### PR TITLE
Nav: wrap strings

### PR DESCRIPTION
**Pull request by sshane**
These are added dynamically so the test didn't catch these.

### CODE REVIEW

**Comment by sshane** [comment]
@eFiniLan @crwusiz can you guys check out traditional Chinese and Korean to make sure these look right?

**Comment by crwusiz** [comment]
> can you guys check out Chinese and Korean to make sure these look right?

eta is 에타 -> 도착예정시간

That's right, but I think it's right to mark "도착" because the width of the letter seems to go over

**Comment by sshane** [comment]
Does it make sense in Korean at all?

**Comment by crwusiz** [comment]
> 

Most navigation systems also indicate the expected arrival time. It doesn't say "에타"

**Comment by crwusiz** [comment]
> Does it make sense in Korean at all?

**Comment by sshane** [comment]
Oh my bad, my English only brain thought what you wrote and what I had was the same 😂 feel free to fix that

**Comment by crwusiz** [comment]
> Oh my bad, my English only brain thought what you wrote and what I had was the same 😂 feel free to fix that

kor translations fix #25105

**Comment by crwusiz** [comment]
i use ubuntu 22
lrelease version 5.15.2

.ts -> .qm

why unit test fail.....?

File: selfdrive/ui/translations/main_zh-CHT.ts [hunk 1]
```diff
@@ -340,6 +340,57 @@
...
+    </message>
+    <message>
+        <location filename="../qt/maps/map.cc" line="632"/>
+        <source>hr</source>
+        <translation>小時</translation>
+    </message>
+    <message>
+        <location filename="../qt/maps/map.cc" line="652"/>
+        <source>km</source>
+        <translation>km</translation>
```
Comment (eFiniLan) [inline selfdrive/ui/translations/main_zh-CHT.ts:363]: `<translation>公里</translation>`

File: selfdrive/ui/translations/main_zh-CHT.ts [hunk 2]
```diff
@@ -340,6 +340,57 @@
...
+    </message>
+    <message>
+        <location filename="../qt/maps/map.cc" line="655"/>
+        <source>mi</source>
+        <translation>mi</translation>
```
Comment (eFiniLan) [inline selfdrive/ui/translations/main_zh-CHT.ts:368]: `<translation>英里</translation>`

**Pull request merged**